pkg/rpm/resolver/base.go

```diff
@@ -14,109 +14,129 @@ import (
 )
 
 const (
-	prepareBaseScriptName = "prepare-base.sh"
-	prepareBaseScriptLog  = "prepare-base.log"
-	baseImageArchiveName  = "sle-micro-base.tar.gz"
-	baseImageRef          = "slemicro"
+	prepareTarballScriptName = "prepare-tarball.sh"
+	prepareTarballScriptLog  = "prepare-tarball.log"
+	tarballName              = "tarball-image.tar.gz"
+	tarballImgRef            = "tarball"
```

Comment:
Can we include more information in the values themselves i.e. what are the tarballs about e.g. `resolver-tarball.log`, `base-resolver-image.log` or something similar? It is completely clear what these are in the context of this package but I'm afraid it'd be ambiguous when users are only looking at the build dir contents.